pkg/testutil/testutil.go

```diff
@@ -224,6 +224,17 @@ func (c *Cmd) AssertNoOut(s string) {
 	c.AssertOutWithFunc(fn)
 }
 
+func (c *Cmd) AssertContains(s string) {
```

Comment:
Actually this is already provided by `AssertOut` (misnomer)

Reply:
ok it's not intuitive